app/views/coronavirus_local_restrictions/devolved_nation_result.html.erb

```diff
@@ -0,0 +1,50 @@
+<% title = t("coronavirus_local_restrictions.results.devolved_nations.heading") %>
+<%= render partial: "meta_tags" %>
+<% content_for :title, title %>
+
+<% content_for :back_link do %>
+  <%= render "govuk_publishing_components/components/back_link", {
+    href: find_coronavirus_local_restrictions_path
+  } %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= tag.div :data => {
+      :module => "coronavirus-track-local-restriction-results",
+      "tracking-label" => "Devolved nation: #{@result.country}"
+    } do %>
+      <%= render "govuk_publishing_components/components/title", {
+        title: title
+      } %>
+      <p class="govuk-body">
+        <%= t("coronavirus_local_restrictions.results.match") %>
+        <strong><%= @result.postcode %></strong>
+        to
+        <strong><%= @result.area_name %></strong>.
+      </p>
```

Comment:
This content is missing from the page: 

`<%= t("coronavirus_local_restrictions.results.devolved_nations.country_rules", country: location.country) %>`